modules/network-observability-without-loki.adoc

```diff
@@ -0,0 +1,19 @@
+// module included in the following assemblies:
+// networking/network_observability/installing-operators.adoc
+
+:_content-type: CONCEPT
+[id="network-observability-without-loki{context}"]
+= Network Observability without Loki
+You can use Network Observability without Loki by omitting the Loki installation steps and only installing Kafka. The following table compares available features with and without Loki:
```

Comment:
```suggestion
You can use Network Observability without Loki by omitting the Loki installation steps and only exporters. The following table compares available features with and without Loki:
```

Exporters can be Kafka / IPFIX and more in future releases :smile_cat:

Reply:
Ah, good clarification. TY!